cmd/object-api-datatypes.go

```diff
@@ -59,6 +59,12 @@ type BucketInfo struct {
 	Created time.Time
 }
 
+// HealInfo - represents erasure coded chunks in disks
+// TBD: Final choice of datatype
+type HealInfo struct {
+	Shards []bool
```

Comment:
``mc`` can use ``HealInfo`` to decide if the object can be healed. Could you give an example of what kind of information would be useful?